CHANGELOG.md

```diff
@@ -1,23 +1,27 @@
-## Version 2.13.0, dev
+## [Unreleased] - 2025-xx-xx
 
 - Upgraded to ASM 9.8
 
-## Version 2.12.0, release 02.02.2025
+
+## [2.12.0] - 2025-02-02
```

Comment:
Or we can inline header links like https://github.com/GradleUp/shadow/pull/1399.

Reply:
There is a lot that could be added - but I am not so eager to maintain the links by hand.

And I am not willing to move further into that direction until this issue got a response here:

https://github.com/olivierlacan/keep-a-changelog/issues/658